aisynphys/database/schema/cortical_location.py

```diff
@@ -0,0 +1,42 @@
+from sqlalchemy.orm import relationship, deferred, sessionmaker, aliased
+from ... import config
+from . import make_table
+from .slice import Slice
+from .experiment import Cell, Experiment
+
+__all__ = ['CellLocation', 'CorticalSite']
+
+CellLocation = make_table(
```

Comment:
Since this table is really about marking positions along the pia-wm lines defined in `CorticalSite`, let's name this `CorticalLocation`.